web/source/osk/visualKeyboard.ts

```diff
@@ -1718,51 +1710,6 @@ namespace com.keyman.osk {
       }
     };
 
-    /**
-     * Wait until font is loaded before applying stylesheet - test each 100 ms
-     * @param   {Object}  kfd   main font descriptor
-     * @param   {Object}  ofd   secondary font descriptor (OSK only)
-     * @return  {boolean}
-     */
-    waitForFonts(kfd, ofd) {
-      let keymanweb = com.keyman.singleton;
-      let util = keymanweb.util;
-
-      let fontDefined = !!(kfd && kfd['files']);
-      kfd = fontDefined ? kfd : undefined;
-
-      let oskFontDefined = !!(ofd && ofd['files']);
-      ofd = oskFontDefined ? ofd : undefined;
-
-      // Automatically 'ready' if the descriptor is explicitly `undefined`.
-      // Thus, also covers the case where both are undefined.
-      var kReady = util.checkFontDescriptor(kfd), oReady = util.checkFontDescriptor(ofd);
-      if (kReady && oReady) {
-        return true;
-      }
-
-      keymanweb.fontCheckTimer = window.setInterval(function () {
-        if (util.checkFontDescriptor(kfd) && util.checkFontDescriptor(ofd)) {
```

Comment:
`util.checkFontDescriptor` is no longer used. `util.checkFont` is no longer used internally either, but has been published as an undocumented API in keymanweb with `isFontAvailable`. Given it was not documented, we should remove it as well!